tests/test_e2e.py

```diff
@@ -66,15 +69,25 @@ def test__e2e__clean_setup__results_are_expected(e2e_test_case, api_client, apps
             "-o",
             f"{result_file.name}",
         ]
-        subprocess.check_output(cmd, timeout=120)
+        try:
+            print(subprocess.check_output(cmd, timeout=120))
+        except subprocess.CalledProcessError as cpe:
+            print(cpe)
     # load contents of result and expected
     result = None
-    with open(result_file_name, "r") as stream:
-        result = yaml.safe_load(stream)
+    try:
+        with open(result_file_name, "r") as stream:
+            result = yaml.safe_load(stream)
+    except OSError:
```

Comment:
In what cases do we expect this exception to appear?

Reply:
When for some kind of reasons `illuminatio` can't be successfully run and won't write the file another way would be to directly fail if the command has an error.